### PR TITLE
deal with calls to /members and /list_source

### PR DESCRIPTION
**Pull request by orangewolf**
this should prevent assets from becoming unreachable.

### CODE REVIEW

File: config/initializers/hyrax.rb [hunk 1]
```diff
@@ -192,6 +192,8 @@
 
   ## Do not alter unless you understand how ActiveFedora handles URI/ID translation
   config.translate_id_to_uri = lambda do |id|
+    return "#{ActiveFedora.fedora.host}#{ActiveFedora.fedora.base_path}/#{::Noid::Rails.treeify(id)}" unless id.match(/^cpb-aacip/)
+    id, tail = id.split('/')
```
Comment (afred) [inline config/initializers/hyrax.rb:196]: For ID's that will have a "tail", are these of the form "cpb-aacip/123-456789"?

Comment (orangewolf) [reply config/initializers/hyrax.rb:196]: the only two I've seen so far are
`cpb-aacip-7fdd93e156c/list_source` or `cpb-aacip-7fdd93e156c/members` but I wanted to cover the generic case so if there was something I wasn't seeing the logs we wouldn't have a sleeper on our hands

Comment (afred) [reply config/initializers/hyrax.rb:196]: Did some digging, and the ID's that come from the pbcore xml in the form `cpb-aacip/123-456789` are getting converted to `cpb-aacip-123-45678` in `PbcoreXMLMapper#normalize_aapb_id` and the `PBCoreXMLMapper` is being used by `PBCoreXMLParser` when initially parsing the record. Approving this

**Review by afred — APPROVED** [review]
👍

**Pull request merged**